lms/templates/ux/reference/course-skeleton.html

```diff
@@ -64,8 +64,30 @@ <h2 class="hd hd-2 page-title">Skeleton Page</h2>
         </div>
     </header>
     <div class="page-content">
-        <h3>This is where the page content belongs</h3>
-        <p>Useful stuff goes here</p>
+        <div class="layout layout-1t2t">
+            <aside class="layout-col layout-col-a" role="complementary" aria-label="Navigation">
+                <h3>Sidebar</h3>
+                <ul>
+                    <li>Item one</li>
+                    <li>Item two</li>
+                    <li>Item three</li>
+                </ul>
+            </aside>
+
+            <main id="main" aria-label="Content" tabindex="-1" class="layout-col layout-col-b">
+                <article tabindex="-1" aria-label="Main Content">
```

Comment:
We already have a Main Content. Let's remove the `tabindex` and the `aria-label`.